tests/scripts/github-action-helper.sh

```diff
@@ -714,7 +714,7 @@ function test_csi_nfs_workload {
 }
 
 function install_minikube_with_none_driver() {
-  CRICTL_VERSION="v1.33.0"
+  CRICTL_VERSION="v1.34.0"
   MINIKUBE_VERSION="v1.36.0"
```

Comment:
update the minikube version? 
```suggestion
  MINIKUBE_VERSION="v1.37.0"
```